ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +36,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *RACDescription(id object) {
+	if (getenv("RAC_DEBUG_SIGNAL_NAMES") != NULL) {
+		if ([object respondsToSelector:@selector(rac_description)]) {
+			return [object rac_description];
+		} else {
+			return [[NSString alloc] initWithFormat:@"<%@: %p>", object.class, object];
```

Comment:
Oops - hadn't thought of that! I never really used to use dot syntax, but I suppose that now I've spent too much time with Swift!